src/Search/search.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { PSATextField } from "../Textfield/textfield";
+import {
+  Box,
+  styled,
+  InputAdornment,
+} from "@mui/material";
+import PropTypes from "prop-types";
+
+const StyledBox = styled(Box)(({ type }) => ({
+  ...(type === "Paper" && {
+    boxShadow: 2,
+    borderRadius: '.2rem',
+    backgroundColor: 'white',
+  }),
+}));
+
+export const PSASearch = ({
+  value,
+  color,
+  handleChange,
+  handleChangeCapture,
+  adornmentContent,
+  inputProps,
+  style,
+  sx,
+  boxProps,
+  label,
+  InputLabelProps,
+  boxType,
+  variant="standard",
+  testId,
+  dataCy,
```

Comment:
remove this since we're not using this as test id anymore

Reply:
this has been removed but i see we are using dataCy on line 67.  This will cause an error if we are no longer passing in this prop.  do we need to remove that line or replace dataCy on that line? 
@BrennenFa @YazidZidane

Reply:
Yes the other `dataCy` on line 67 should also be removed.